infra/urls.py

```diff
@@ -0,0 +1,18 @@
+'''Register URL routes in auth module.'''
+from rest_framework import routers
+from rest_framework_jwt.views import verify_jwt_token, obtain_jwt_token
+from django.urls import path
+
+import infra.views
+
+
+router = routers.SimpleRouter()
+router.register(r'operationlog', infra.views.OperationLogViewSet)
```

Comment:
```suggestion
```
OperationLog should not be handled by users, instead we handle it behind the scene, so we should not expose routes for this.